editing_and_imputation/imputation/general/technical_specification.md

```diff
@@ -222,7 +234,8 @@ to the construction link
 
 In link calculations, only responders present in the same group in both
 target and predictive periods shall be used to calculate the ratios. These
-pairs of responders are referred to as matched pairs.
+pairs of responders are referred to as matched pairs. Non responders
+with a manual contruction value are excluded from link calculaion.
```

Comment:
Minor typo - calculation

Reply:
Removed this sentence as previous statement `**only responders** present in the same group in both
target` will cover manual construction contributor as well.